assets/elm/src/Main.elm

```diff
@@ -51,29 +50,35 @@ init config location =
 
 update : Msg -> AppModel -> ( AppModel, Cmd Msg )
 update msg model =
```

Comment:
Sorry this is a git mess up: only things actually changed are:
1. removed debug statement
2. added MoveUp message branch